src/app/features/settings/go-pro/go-pro-media-viewer-with-native-player/go-pro-media-viewer-with-native-player.component.ts

```diff
@@ -33,24 +32,24 @@ export class GoProMediaViewerWithNativePlayerComponent
   ngOnInit(): void {
     if (!this.mediaFile?.url) return;
     if (this.platform.is('hybrid')) {
-      PreviewVideo.playFullScreenFromRemote({ url: this.mediaFile.url });
-      this.onIOSPlayerDismissedListener = PreviewVideo.addListener(
-        'iosPlayerDismissed',
-        (_info: any) => this.navController.back()
-      );
+      // PreviewVideo.playFullScreenFromRemote({ url: this.mediaFile.url });
+      // this.onIOSPlayerDismissedListener = PreviewVideo.addListener(
+      //   'iosPlayerDismissed',
+      //   (_info: any) => this.navController.back()
+      // );
     }
   }
 
   ngOnDestroy(): void {
     if (this.platform.is('hybrid')) {
-      PreviewVideo.stopFullScreen();
-      this.onIOSPlayerDismissedListener?.remove();
+      // PreviewVideo.stopFullScreen();
+      // this.onIOSPlayerDismissedListener?.remove();
     }
   }
 
   dismiss() {
     if (this.platform.is('hybrid')) {
-      PreviewVideo.stopFullScreen();
+      // PreviewVideo.stopFullScreen();
```

Comment:
Could you elaborate on why comment out these codes?

Reply:
@shc261392, I changed [preview-video](https://github.com/numbersprotocol/preview-video) pluging that introduced breaking chnages. Since version [0.0.2](https://github.com/numbersprotocol/preview-video/releases) preview-video plugin respects HTML scrolling and it does not have `PreviewVideo.stopFullScreen()` methods anymore.

Since go-pro related features were not main task of the spint I just decided to comment out temporarely and when we return to go-pro related features I can easily recall and develop related features.

@shc261392 please let me know if I should completely remove the commends and keep go-pro-media-viewer-with-native-player.component.ts empty.

Reply:
Thanks for the explanation. I think it's fine to leave it as it is.